buildsystem/pxdgen.py

```diff
@@ -366,42 +352,23 @@ def parse_args():
     import argparse
 
     cli = argparse.ArgumentParser()
-    cli.add_argument('files', nargs='*', metavar='HEADERFILE',
-                     help="input files (usually cpp .h files).")
-    cli.add_argument('--file-list',
-                     help="semicolon-separated list of input files.")
-    cli.add_argument('--ignore-timestamps', action='store_false',
-                     help="force generating even if the output file is already"
-                          "up to date")
-
-    args = cli.parse_args()
+    cli.add_argument("input", help=("input file (usually cpp .h files)."))
+    cli.add_argument("output", help=("The output is stored in this file."))
 
-    if args.file_list:
-        file_list = open(args.file_list).read().strip().split(';')
-    else:
-        file_list = []
-
-    from itertools import chain
-    args.all_files = chain(args.files, file_list)
-
-    return args
+    return cli.parse_args()
 
 
 def main():
     """ CLI entry point """
     args = parse_args()
     cppdir = CWD + "/libopenage"
 
-    for filename in args.all_files:
```

Comment:
We added support for multi file conversion because booting up the python VM for each file is very slow. Batch support is much faster for many files.